mobify/platform-scaffold
# Add delay to modal dismiss

Modal manager removes modal right away when user tries to close the modal, that stops the close animation. This PR introduces a delay for removing modals.

It also introduces a way to unify the transition time for modals throughout a project:
`<ModalManager duration={200} />`

as well as set individual modal transition time (inside ModalManger):
`[MODAL.NAVIGATION_MODAL]: {content: <Navigation />, customDuration: 2000}`


 **JIRA**: none
 **Linked PRs**: none

## Changes
- Add delay for close animation
- Unify transition time for all modals in Modal Manager

## Todos
~~- [ ] (if applicable) analytics events instrumented to measure impact of change~~

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open and close modals to verify the animation is working
